packages/firebase_core/firebase_core_web/lib/src/firebase_core_web.dart

```diff
@@ -35,9 +35,6 @@ typedef EnsurePluginInitialized = Future<void> Function()?;
 class FirebaseCoreWeb extends FirebasePlatform {
   static Map<String, FirebaseWebService> _services = {
     'core': FirebaseWebService._('app', override: 'core'),
-    'app-check': FirebaseWebService._('app-check', override: 'app_check'),
```

Comment:
This isn't going to work. We still need to rename those libraries, just not add them as a dependency.